DSA-Campaign-Uplift-Estimation/src/main/webapp/Home/script-home.js

```diff
@@ -1,13 +1,30 @@
-// Copyright 2019 Google LLC
+//Copyright 2020 Google LLC
 //
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
+//Licensed under the Apache License, Version 2.0 (the "License");
+//you may not use this file except in compliance with the License.
+//You may obtain a copy of the License at
 //
-//     https://www.apache.org/licenses/LICENSE-2.0
+//    https://www.apache.org/licenses/LICENSE-2.0
 //
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+//Unless required by applicable law or agreed to in writing, software
+//distributed under the License is distributed on an "AS IS" BASIS,
+//WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//See the License for the specific language governing permissions and
+//limitations under the License.
+
+function getLogsStatus() {
+
+    fetch('/userapi').then(response => response.json()).then(logStatus => {
+        const link = document.getElementById("logout-link");
+
+        if(logStatus.Bool) {
+
+            console.log('User is Logged In');
+            link.href = logStatus.Url;
+        }
+        else {
+            console.log('User is not Logged In');
+            link.href = '../index.html';
+        }
+    });
+}
```

Comment:
This code seems to be repeated a lot. Is there any way we can have a single definition each js file can reference.

Reply:
I was think of moving this to the navbar-script.js in a future CL when navbar-script.js is called on in html so we don't have to continuously use this in other script files. The navbar only utilizes this too.

Reply:
Can we do this now? Better to get rid of code duplication now than later.

Reply:
Yes and Done.